auth/handlers/oauth2.go

```diff
@@ -186,3 +192,46 @@ func getToken(res *http.Response) (GitHubAccessToken, error) {
 
 	return token, fmt.Errorf("no body received from server")
 }
+
+func getUserOrganizations(name string) (string, error) {
+
+	organizations := []Organization{}
+	apiURL := fmt.Sprintf("https://api.github.com/users/%s/orgs", name)
```

Comment:
You need to use the token, if you don't use the token you won't get to see the organisations that have access granted. You'll also use up the API tokens since they only allow 50 per hour unauthorised.